docs/logging/05-02-production-profile.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Logging production profile
+type: Configuration
+---
+## Overview
+
+To use production-ready Logging, you can install Kyma with the Logging production profile. Higher memory limits set for Loki and FluentBit logging solutions ensure stable log processing in a production environment.
```

Comment:
We should get more specific in the introduction, something like that:

It is a production-ready profile under restriction of using the out-of-the box solution.
Loki will not be high-available as it cannot scale out with the default persistence. In order to scale it out it requires an external storage for chunk and index store like GCS and Bigtable. 
Loki in the production profile has a retention period of 1 day and is meant to be used as a convenient way of accessing logs via the Kyma Console.

Reply:
👍  I will add this.

Reply:
Done.